views/welcome_wizard.html

```diff
@@ -54,3 +55,12 @@
 'gotIt': _x('Got it!', 'A label on a button'),
 }) %>
 <% endblock %>
+
+<% block after_javascript %>
+  <script type="text/javascript">
+    console.log(window.Beacon)
+    if (window.Beacon) {
+      window.Beacon('destroy');
+    }
+  </script>
+<% endblock %>
```

Comment:
There is a `console.log()` here. But also - what is this whole block actually for?

Reply:
oops! this block was only for debugging, I removed it.